docs/how-to-guides/feathr-configuration-and-env.md

```diff
@@ -77,9 +77,9 @@ Feathr will get the configurations in the following order:
 | REDIS_PASSWORD                                        | Password for the Redis cluster.                                                                                                                                                                                                                                                    | Required if using Redis as online store.                                  |
 | FEATURE_REGISTRY__API_ENDPOINT                        | Specifies registry endpoint.                                                                                                                                                                                                                                                       | Required if using registry service.                                       |
-| FEATURE_REGISTRY__PURVIEW__PURVIEW_NAME               | Configure the name of the purview endpoint.                                                                                                                                                                                                                                        | Required if using Purview directly without registry service. Deprecate soon, see [here](#deprecation) for more details.|
-| FEATURE_REGISTRY__PURVIEW__DELIMITER                  | See [here](#FEATURE_REGISTRY__PURVIEW__DELIMITER) for more details.                                                                                                                                                                                                                | Required if using Purview directly without registry service. Deprecate soon, see [here](#deprecation) for more details.|
-| FEATURE_REGISTRY__PURVIEW__TYPE_SYSTEM_INITIALIZATION | Controls whether the type system (think this as the "schema" for the registry) will be initialized or not. Usually this is only required to be set to `True` to initialize schema, and then you can set it to `False` to shorten the initialization time.                          | Required if using Purview directly without registry service. Deprecate soon, see [here](#deprecation) for more details.|
+| FEATURE_REGISTRY__PURVIEW__PURVIEW_NAME  (Deprecated)             | Configure the name of the purview endpoint.                                                                                                                                                                                                                                        | Required if using Purview directly without registry service. Deprecate soon, see [here](#deprecation) for more details.|
```

Comment:
Please change Deprecated to Deprecated Soon, I think @xiaoyongzhu still would like to keep direct purview connect for 1 or 2 releases

Reply:
Changed.